### PR TITLE
Added separate RabbitMQ service

### PR DESCRIPTION
**Pull request by L4v**

### CODE REVIEW

**Review by st3fan7 — APPROVED** [review]

File: RabbitMQService/RabbitMQHosedService.cs [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                    }
+                };
+
+                channel.BasicConsume(queue: "psw-queue",
+                                     autoAck: false,
+                                     consumer: consumer);
+                */
+                return StartAsync(cancellationToken);
+            }
+        }
```
Comment (jelena-vlajkov) [inline RabbitMQService/RabbitMQHosedService.cs:98]: Delete commented lines.

File: RabbitMQService/RabbitMQHosedService.cs [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                        }
+                        catch
+                        {
+                            channel.BasicReject(reply.DeliveryTag, true);
+                        }
+                    }
+                    else
+                        recive = false;
+                }
+
```
Comment (jelena-vlajkov) [inline RabbitMQService/RabbitMQHosedService.cs:72]: Try to refactor this method in less lines.

File: RabbitMQService/RabbitMQService.cs [hunk 3]
```diff
@@ -20,46 +21,65 @@ public class RabbitMQService : BackgroundService
...
+            //    {
+            //        var body = reply.Body.ToArray();
+            //        var msg = Encoding.UTF8.GetString(body);
+
+            //        channel.BasicAck(reply.DeliveryTag, false);
+            //    }
+            //    else
+            //        recive = false;
+            //}
+
```
Comment (jelena-vlajkov) [inline RabbitMQService/RabbitMQService.cs:60]: Also delete these commented lines.

**Review by jelena-vlajkov — APPROVED** [review]
Delete commented lines and refactor big methods.

**Pull request merged**